livechat/reports/base.py

```diff
@@ -0,0 +1,52 @@
+''' Base class that allows retrieval of client for specific Reports
```

Comment:
Base module/Module with base class*

Reply:
Fixed in 3c5b25632a401cd9fc34a0137405e70c83b297a9.